relay-general/src/protocol/contexts/browser.rs

```diff
@@ -21,3 +24,229 @@ impl BrowserContext {
         "browser"
     }
 }
+
+impl FromUserAgentInfo for BrowserContext {
+    fn from_client_hints(client_hints: &user_agent::ClientHints) -> Option<Self> {
+        let browser = Browser::from_ua_client_hint(client_hints.sec_ch_ua?)?;
+        let version = client_hints.sec_ch_ua_full_version?.to_owned();
```

Comment:
`sec-ch-ua-full-version` seems to be [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA-Full-Version). The [significant version](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA#syntax) is probably more relevant to our users than the full version anyway, so should we just parse that from the `sec-ch-ua` hint?